src/FacebookApi.cpp

```diff
@@ -94,13 +96,13 @@ String FacebookApi::sendGetToFacebook(String command) {
 }
 
 String FacebookApi::getFriends(){
-	String command="/v2.9/me/friends?access_token="+_accessToken;
+	String command = "/v2.9/me/friends?access_token="+_accessToken;
 	//Serial.println(command);
 	return sendGetToFacebook(command);  //recieve reply from facebook
 }
 
-String FacebookApi::extendAccessToken(String appId, String appSecret){
-	String command="/oauth/access_token?client_id="+ appId +"&client_secret=" + appSecret + "&grant_type=fb_exchange_token&fb_exchange_token="+_accessToken;
+String FacebookApi::extendAccessToken() {
```

Comment:
Same as above, it is a breaking change. 

You could either default them or create a second extendAccessToken, one that takes two strings and one that doesn't. Then one that doesn't take parameters could call the one with the string passing in the class properties passed into the constructor.
```
String FacebookApi::extendAccessToken(){
    extendAccessToken(_appId , _appSecret );
}
```